.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```

Can we do `weekly`? I'm a big fan of dependabot but I don't want to get spammed by updates every day.

```suggestion
      interval: weekly
```

---

Since we're only using major version on actions, I think it would be better to keep `daily`, so we can take action asap

---

Fair enough.